scripts/es-commercial-docker.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
do we need this?

can we not just alter the other one to accept CLI paremeters, and if the `elastic_password` is set then we append all the xpack stuff otherwise we execute as normal?

Reply:
I'm using it in my machine to run quick test :)